impl/src/main/java/org/jboss/cdi/tck/tests/interceptors/definition/interceptorOrder/InterceptorOrderTest.java

```diff
@@ -59,20 +59,4 @@ public void testInterceptorsCalledInOrderDefinedByBeansXml(Foo foo) {
         assertEquals(sequence.get(1), FirstInterceptor.class.getName());
     }
 
-    @Test(dataProvider = ARQUILLIAN_DATA_PROVIDER)
-    @SpecAssertion(section = ENABLED_INTERCEPTORS, id = "g")
-    public void testInterceptorsInvocationOrder(AccountTransaction transaction) {
-
-        assertNotNull(transaction);
-        ActionSequence.reset();
-
-        transaction.execute();
-
-        List<String> sequence = ActionSequence.getSequenceData();
-        assertEquals(sequence.size(), 4);
-        assertEquals(sequence.get(0), AnotherInterceptor.class.getName());
-        assertEquals(sequence.get(1), TransactionalInterceptor.class.getName());
-        assertEquals(sequence.get(2), Transaction.class.getName());
-        assertEquals(sequence.get(3), AccountTransaction.class.getName());
-    }
```

Comment:
I think this test file doesn't need to be split and should be moved to `full` entirely. The 1st test (kept in this file in this PR) tests `beans.xml` enablement, the other test (moved to `full` in this PR) uses both `@Interceptors` and `@AroundInvoke` on bean class.